### ISSUE TITLE
Braille fonts not rendering correctly

### ISSUE THREAD

**Issue by lands39** [aristocratos/btop#971]
Issue: Braille fonts not rendering correctly

Reproduce: Install btop on any linux distro and use braille. 

Expected behavior: Braille is supposed to look like dots, not hashtags. just found this out today, i didn't know it wasn't working right. 

Screenshot: 

![image](https://github.com/user-attachments/assets/0d9eec37-897a-4700-8bbf-cc559982cba8)

Distro: Arch, but this affects all machines i use, including those on debian, raspberry pi OS and armbian. 
Font: braille

**Comment by CryZo** [comment]
@lands39 I also had problems on Arch with braille fonts. I don't think that btop is the Problem here. It's just the default braille font. This fixed it for me:
https://wiki.archlinux.org/title/Fonts#Braille_font_not_displaying_correctly_inside_terminals

TL;DR:
1. Uninstall `gnu-free-fonts`.
2. Install a font containing braille characters you like more - like [`ttf-ubraille`](https://aur.archlinux.org/packages/ttf-ubraille/) from AUR for example.

**Comment by cprn** [comment]
Also, if you're using `urxvt`, remember to:

1. put the font in your `~/.Xresources`, e.g.:
```
URxvt.font: xft:JetBrainsMono Nerd Font:size=14,xft:Braille:size=24
```
2. reload them:
```sh
xrdb ~/.Xresources
```
3. restart terminal

This issue can be closed, I believe. It's not `btop`'s fault.
